### DISCUSSION TITLE
uvicorn url is not working locally. http://127.0.0.1:8000

### DISCUSSION THREAD

**Discussion by venkataseetharaman** [fastapi/fastapi#10663] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
import uvicorn
import logging

app = FastAPI()
logging.info('Created FASTAPI instance')

@app.get("/")
async def root():
    return {"message" : "Hello World!!!"}
logging.info('Method is called')
```

### Description

I am new to FASTAPI & Python development. Created my first FASTAPI with above code.
Tried uvicorn main:app --reload to run the uvicorn server. It has started without any error.
Upon hitting the url http://127.0.0.1:8000 in web browser( Chrome & Postman), it's been spining and no results.

Options tried
1. Added inbound rule in firewall
2. I have stopped the firewall and anti virus.

Nothing worked for me

### Operating System

Windows

### FastAPI Version

0.1.0

### Pydantic Version

1

### Python Version

3.9

**Comment by hockeyiscool19** [comment]
Yes, I have tried the same! Postman did not work for me either, but FastApi includes its own user interface to test endpoints, Try http://127.0.0.1:8000/docs#/. If you use this url, you will see the different routes. Click "try it out." Then click execute. LMK if this works!

**Reply by venkataseetharaman** [reply]
Thanks for the response. I was not able to browse this URL as well. What i did is, I installed Python 3.12 version and tried. it worked for me. I was using 3.9 version earlier.

**Reply by hockeyiscool19** [reply]
Great! Let's gooo.

**Comment by hasansezertasan** [comment, marked as answer]
Can you please try visiting [http://localhost:8000](http://localhost:8000) instead of `127.0.0.1`?

I'm on a windows machine either, and `127.0.0.1` doesn't work for me sometimes, IDK why, never wondered. Just use `localhost`...

**Reply by noorzaibchohan** [reply]
Thankyou Mate. This works but not all time. People can refer to the  Github Co-Pilot for potential errors in codespace causing server errors. for me it was just a line change and everything worked fine.

**Reply by Ajibola-Oki-Mappa** [reply]
what was it? I'm going through the same thing

**Reply by Tsuraj56** [reply]
kill all instances of python using task manager. it worked for me

**Comment by ranch-hands** [comment]
if u r using colab or jupyter notebook u shd use ngrok to use the url as an external port.
